tests/src/Functional/ContentEventRenderTest.php

```diff
@@ -130,6 +131,12 @@ public function testEventPage(): void {
       trim($content_banner->filter('.oe-sc-event__oe-summary')->text())
     );
 
+    // Assert registration button.
+    $link = $crawler->filter('div.container > div.row > div.col-8 > a[target="_blank"]');
+    $this->assertCount(1, $link);
```

Comment:
below also assert that there is a button inside the link, not just the button text